platform/mikro-e/apps/test-button/Makefile

```diff
@@ -1,6 +1,9 @@
 CONTIKI_PROJECT = test-button
-all: $(CONTIKI_PROJECT)
-
+ifndef TARGET
```

Comment:
Either if all these sample/test apps are not dependent on platform then it should not be under mikro-e, but if its dependent, then why can't we have TARGET=mikro-e unconditionally?